docs/source/models/risk_exposures/child_growth_failure/wasting/wasting_state_exposure/index.rst

```diff
@@ -208,6 +208,8 @@ Given the following equations:
 
 2. :math:`RR = p_\text{cat1,LBW} / p_\text{cat1,ABW}` 
 
+We can then solve for the ABW and LBW probabilities of initialization into wasting categories 1 and 2. We then assume that the difference between the ABW and LBW probabilities for categories 1 and 2 with the population-level probabilities is equally distributed amongst categories 3 and 4.
```

Comment:
We have a 4 category exposure model for all age groups, but our effect of LBWSG is based on a 2-category effect of wasted (cat1 and 2) vs. non-wasted (cat3 and 4). We keep the 4 category exposure model, but apply the effect of LBWSG in a way that cat1 and 2 change by the same proportion and the same for cat3 and 4.

Does that clarify?

Reply:
First, we apply the RR of LBWSG-> wasting to categories 1 and 2. Then, we reapportion the exposure so that the total across all categories is equal to 1 and the exposure of cat3 and cat4 scale in equal proportions.

Reply:
Just sent you an email with an excel file that shows and example calculation

Reply:
@aflaxman any remaining concerns?

Reply:
No concerns.  Thanks for the spreadsheet demonstration, that made it clear.